torch/_functorch/partitioners.py

```diff
@@ -96,10 +163,7 @@ def _extract_graph_with_inputs_outputs(joint_graph, inputs, outputs):
         env[node] = new_node
 
     for node in joint_graph.nodes:
-        if node in env:
-            # Node must be one of our inputs. (Any member of env which wasn't an
-            # input to start must have been created by this loop and won't be in
-            # joint_graph.nodes).
+        if node in inputs:
```

Comment:
Why this? Using `node in env` reduced the compile time for a model by 20%.

Reply:
This might be the result of a rebase issue I had 🤔

Reply:
Changed back.